src/Lean/MetavarContext.lean

```diff
@@ -240,10 +240,12 @@ def LocalInstances.erase (localInsts : LocalInstances) (fvarId : FVarId) : Local
   | some idx => localInsts.eraseIdx idx
   | _        => localInsts
 
```

Comment:
It would be great to get the docstrings for `MetavarContext.lean` in a separate PR.

Reply:
#1613